Reference for stage 3 temporal (#37344)
mdn/content: files/en-us/web/javascript/reference/global_objects/temporal/zoneddatetime/toplaindatetime/index.md

---
title: Temporal.ZonedDateTime.prototype.toPlainDateTime()
slug: Web/JavaScript/Reference/Global_Objects/Temporal/ZonedDateTime/toPlainDateTime
page-type: javascript-instance-method
browser-compat: javascript.builtins.Temporal.ZonedDateTime.toPlainDateTime
---

{{JSRef}}

The **`toPlainDateTime()`** method of {{jsxref("Temporal.ZonedDateTime")}} instances returns a new {{jsxref("Temporal.PlainDateTime")}} object representing the date and time portions of this date-time. Only the time zone information is removed.

> [!WARNING]

> > After a `Temporal.ZonedDateTime` is converted to `Temporal.PlainDateTime`, it will no longer be aware of its time zone. This means that subsequent operations like arithmetic or with will not adjust for DST and may not yield the same results as equivalent operations with `Temporal.ZonedDateTime`. However, unless you perform those operations across a time zone offset transition, it's impossible to notice the difference. Therefore, be very careful when performing this conversion because subsequent results may look correct most of the time while failing around time zone transitions like when DST starts or ends.

Reviewer: Minor one if you like:

```suggestion
> After a `Temporal.ZonedDateTime` is converted to `Temporal.PlainDateTime`, it's no longer time zone aware. Subsequent operations like arithmetic or `with` operations will not adjust for DST and may not yield the same results as equivalent operations with `Temporal.ZonedDateTime`. However, unless you perform those operations across a time zone offset transition, it's impossible to notice the difference. Be very careful when performing this conversion because subsequent results are probably correct most of the time, but may be incorrect across time zone transitions like when DST starts or ends.
```
